OSSM-3251: Add cluster-wide deployment to Service Mesh docs (#58539)
openshift/openshift-docs: modules/ossm-deploy-cluster-wide-mesh.adoc

// Module included in the following assemblies:
// * service_mesh/v2x/ossm-deployment-models.adoc

:_content-type: CONCEPT
[id="ossm-deploy-cluster-wide-mesh_{context}"]
= Cluster-Wide (Single Tenant) mesh deployment model

> A cluster-wide deployment contains a Service Mesh Control Plane that monitors resources for an entire cluster. This more closely matches Istio functionality, where the control plane uses a single query across all namespaces to monitor Istio and Kubernetes resources. As a result, cluster-wide deployments decrease the number of requests sent to the API server.

Reviewer: ```suggestion
A cluster-wide deployment contains a Service Mesh Control Plane that monitors resources for an entire cluster. This [what?] closely resembles Istio functionality in that the control plane uses a single query across all namespaces to monitor Istio and Kubernetes resources. As a result, cluster-wide deployments decrease the number of requests sent to the API server.
```
- same as earlier similar comment

Author: Done

> Similar to Istio, a cluster-wide mesh includes namespaces with the `istio-injection=enabled` namespace label by default. You can change this by modifying the `spec.labelSelectors` field of the `ServiceMeshMemberRoll` resource. 

Reviewer: ```suggestion
Similar to Istio, a cluster-wide mesh includes namespaces with the `istio-injection=enabled` namespace label by default. You can change this label by modifying the `spec.labelSelectors` field of the `ServiceMeshMemberRoll` resource. 
```

Author: Done
